todo_app/app.py

```diff
@@ -3,40 +3,284 @@
 
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
-from todo_app.data.session_items import get_item
-from todo_app.data.session_items import save_item
+from todo_app.data.session_items import get_items, add_item, get_item, save_item
+import requests
+import json
+import os
+import logging
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
+#@app.route('/')
+
+#r= requests.get("https://api.trello.com/1/members/me/boards?fields=name,url&key=77f5ab920b78a542ed2fb7501190d22d&token=1b81c916acc9144aa57afc7decdd1249d033e0bf52d300c5587dd38aa3acc846")
```

Comment:
Some of these comments have your Trello credentials in them, so anybody could find your API key/token. We should remove these, but also revoke the credentials within Trello and set up new ones so that we don't need to worry about them sticking about in the git history

Even if they didn't have credentials, it would be worth removing all of these lines to make the code easier to follow for future

Reply:
I tried to edit the file but Github is asking me to be on the branch, when I switch to the branch it's letting me edit only the final commit.Is there another way to delete them?